airflow/models/dag.py

```diff
@@ -1506,6 +1505,17 @@ def safe_dag_id(self):
         return self.dag_id.replace('.', '__dot__')
 
     def get_dag(self):
+        # TODO: Resolve this in upstream by storing relative path in db (config driven)
+        try:
+            # fix for manual trigger, as db stores scheduler path which is different from webserver
```

Comment:
Change to: # Fix for DAGs that are manually triggered in the UI, as the DAG path in the DB is stored by the scheduler which has a different path than the webserver due to absolute paths in aurora including randomly generated job-specific directories. Due to this the path the webserver uses when it tries to trigger a DAG does not match the existing scheduler path and the DAG can not be found.